README.md

```diff
@@ -1,5 +1,15 @@
 # GOV.UK coronavirus - find support
 
+⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️⚠️
+
+<p style="text-align: center">This repository is the old Coronavirus Find Support service - https://find-coronavirus-support.service.gov.uk.</p>
+
+<p style="text-align: center; font-weight: bold;">It is no longer deployed or in active development.</p>
+
+<p style="text-align: center">The service was migrated to [Smart Answers](https://github.com/alphagov/smart-answers) and is now available here: https://www.gov.uk/find-coronavirus-support</p>
```

Comment:
Looking at the output - it looks like the "Smart Answers" link isn't rendering as intended. I think the markdown syntax isn't working withing a `<p>` tags. See https://github.com/alphagov/govuk-coronavirus-find-support/blob/3cb47d5df0b2833d36a9cf80a3fdf0d3d29b4e57/README.md

Reply:
Do we even need the <p> tags?

Reply:
@reggieb - thanks - fixed